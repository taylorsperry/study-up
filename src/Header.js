import React from 'react';
import './scss/_Header.scss';

const Header = props => {
    return (
        <section className="header">
            <h1 className="logo">StudyUp</h1>
        </section>
    );
};

export default Header;