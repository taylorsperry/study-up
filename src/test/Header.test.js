import React from "react";
import Header from "../Header";
import { shallow } from "enzyme"

describe("Header", () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallow(<Header />)
  })

  it("should match snapshot when all data is passed correctly", () => {
    expect(wrapper).toMatchSnapshot();
  });
})