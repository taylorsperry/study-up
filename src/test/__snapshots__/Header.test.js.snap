// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header should match snapshot when all data is passed correctly 1`] = `
<section
  className="header"
>
  <h1
    className="logo"
  >
    StudyUp
  </h1>
</section>
`;
